flaml/automl/task/generic_task.py

```diff
@@ -0,0 +1,849 @@
+import logging
```

Comment:
Most code in this file is moved from automl.py. I didn't verify whether there is any unnecessary change when moving. I'll appreciate if another reviewer could check it.